src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -102,7 +103,13 @@ const Body = BaseForm.extend({
           }
         })
         .catch(() => {
-          Logger.error('Something unexpected happened during device probing.');
```

Comment:
I'm wondering if this would work slightly better?
- logger error detail and increase failed port count in `catch` callback
- cancel polling in `finally` callback

Reply:
@haishengwu-okta I tried and noticed some issues regarding you advice:
1. Since we iterate through several ports, this finally block must be outside of `.forEach`
2. If 1 is true, it's working good with the failfast scenario. But for successful connection with any port, there will still be a `/poll/cancel` api call.
3. Maybe we can refactor the logic to follow the advice, but i think that's out of scope on this PR and is kind of risky.
@jhe-okta Can you also take look here?